test/15_extras.jl

```diff
@@ -87,6 +87,27 @@ const ≅ = isequal
     @test isa(x, CategoricalMatrix{Union{String, T}})
     @test isordered(x)
     @test levels(x) == ["[-2.134, 3.0)", "[3.0, 12.5)"]
+
+    labels = 0:2:8
+    x = @inferred cut(Vector{Union{T, Int}}([1,2,3,4,5,6,7,8]), 0:2:10; labels)
```

Comment:
To support Julia 1.0:
```suggestion
    x = @inferred cut(Vector{Union{T, Int}}([1,2,3,4,5,6,7,8]), 0:2:10, labels= labels)
```